graphics/potrace/Portfile

```diff
@@ -23,8 +23,8 @@ long_description    Potrace is a utility for tracing a bitmap, which means, \
 homepage            http://potrace.sourceforge.net/
 master_sites        sourceforge:project/potrace/${version}/
 
-checksums           rmd160  3693c10fae055c36b9728a21f3d2739a3fa8df74 \
-                    sha256  a9b33904ace328340c850a01458199e0064e03ccaaa731bc869a842b1b8d529d
+checksums           rmd160  7301ae2d2ae88e5f9825d7655e046d98a8083a6c \
+                    sha256  be8248a17dedd6ccbaab2fcc45835bb0502d062e40fbded3bc56028ce5eb7acc
```

Comment:
Please add the size checksum

Reply:
done. I used `port bump` maybe it should check and add if not found. Same if found md5, sha1

Reply:
I agree it would be nice if port bump would check for required/obsolete checksum entries and adjust accordingly. Can I suggest you submit a trac ticket suggesting this ?

Reply:
Ok. I will open a trac ticket about the topic.

Thanks for all merges!